source/manual/encrypted-hiera-data.html.md

```diff
@@ -150,17 +150,27 @@ recipient file pertains to.
 
 ### What to do when someone joins
 
-1. Ask the joiner to [create a GPG key](create-a-gpg-key.html) and upload it
-   to a public key server (such as <https://pgp.mit.edu/>). You may need to [manually import the key](#manually-import-the-key) to your GPG Keychain.
-2. Get the fingerprint of the new GPG key by running `gpg --fingerprint`.
-3. Add the joiners's GPG fingerprint to the recipient files
+1. Ask the joiner to [create a GPG key](create-a-gpg-key.html).
+
+1. Ask them to export and send you their public key.
+
+   ```
+   gpg --export -a --output your_name.asc your.email@digital.cabinet-office.gov.uk
+   ```
```

Comment:
```suggestion
   Send the `.asc` file you created as part of step 1.
```

Reply:
Thanks, I've updated a couple of the steps here to make them clearer.